src/Analysis/Ast/Impl/Modules/PythonModule.cs

```diff
@@ -376,7 +381,13 @@ internal PythonModule(ModuleCreationOptions creationOptions, IServiceContainer s
                     throw new OperationCanceledException();
                 }
                 _ast = ast;
-                _parseErrors = sink.Diagnostics;
+                _parseErrors = sink?.Diagnostics ?? Array.Empty<DiagnosticsEntry>();
+
+                // Do not report issues with libraries or stubs
+                if (sink != null) {
+                    _diagnosticsService?.Replace(Uri, _parseErrors);
```

Comment:
I'm assuming this is valid because at this point the module is going to be re-analyzed, so we wouldn't be throwing away any diagnostics we actually need to keep?

Reply:
Correct. I am adding reporting of imports in the next PR and that will be combining parse + analysis diagnostics.